assignments/callbacks.js

```diff
@@ -38,28 +38,59 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
   console.log(test2); // "this Pencil is worth a million dollars!"
 */
 
+// function getLength(items, callback) {
+//   return callback(items);
+// }
 
-function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
-}
+// const callback = items => {
+//   return items.length;
+// };
 
-function last(arr, cb) {
-  // last passes the last item of the array into the callback.
-}
+// console.log(getLength(items, callback));
 
-function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
-}
+// function last(arr, callback) {
+//   // last passes the last item of the array into the callback.
+//   return callback(arr[arr.length-1]);
+// }
 
-function multiplyNums(x, y, cb) {
-  // multiplyNums multiplies two numbers and passes the result to the callback.
-}
+// const callback = function callback(items) {
```

Comment:
The reason why an error is popping up when all of this code is uncommented is beacuse you have 
```
const callback = 
```

typed twice, and the code only lets you declare a constant twice. The fix for this is pretty simple- give the second callback function a different name, like `callback2`.